prow/oss/cluster/cluster.yaml

```diff
@@ -94,7 +85,7 @@ metadata:
     kubernetes.io/ingress.global-static-ip-name: "oss-prow"
     kubernetes.io/ingress.class: "gce"
     kubernetes.io/tls-acme: "true"
-    networking.gke.io/managed-certificates: oss-prow-knative-dev,oss-prow-private-knative-dev
+    networking.gke.io/managed-certificates: oss-prow-knative-dev
 spec:
   rules:
   - host: oss-prow.knative.dev
```

Comment:
We should probably also temporarily remove the oss-prow-private.knative.dev host rule.

Reply:
removed